eoverdracht/leveranciersspecificatie.md

```diff
@@ -283,11 +282,9 @@ De verschillende processtappen in de verpleegkundige overdracht vereisen een bep
 
 Het concept van een grondslag verbindt welke ontvangende partij toegang krijgt tot welke gegevens bij welke bronhouder inzake welke patiënt. En hoewel er voor het aanmeldbericht dus geen valide grondslag is om persoonsgegevens te verwerken, is het voor de implementatie wel handig om beide berichten via eenzelfde mechanisme toegankelijk te maken. De grondslag is onderdeel van een speciaal autorisatie record, deze is beschreven in [RFC014](https://nuts-foundation.gitbook.io/drafts/rfc/rfc014-authorization-credential).
 
-In het kader van de verpleegkundige overdracht zal de grondslag een verwijzing bevatten naar het aanmeldbericht. Dit is dan ook meteen de scope waartoe de ontvanger van de grondslag toegang heeft.
+In het kader van de verpleegkundige overdracht zal de autorisatie een verwijzing bevatten naar het aanmeldbericht. Dit is dan ook meteen de scope waartoe de ontvanger van de grondslag toegang heeft.
 
-Naast het eerder genoemde voordeel van herbruikbaarheid is een ander voordeel van het gebruik van grondslagen dat de patiënt dit ook inzichtelijk zou kunnen krijgen in een PGO. De patiënt zou op basis van de grondslagen een overzicht kunnen krijgen van wie zijn of haar gegevens kan inzien, en waarom.
-
-Er wordt momenteel gewerkt aan een specificatie voor het automatisch uitwisselen van grondslagen.
+Naast het eerder genoemde voordeel van herbruikbaarheid is een ander voordeel van het gebruik van autorisaties dat de patiënt dit ook inzichtelijk zou kunnen krijgen in een PGO. De patiënt zou op basis van de autorisaties een overzicht kunnen krijgen van wie zijn of haar gegevens kan inzien, en waarom.
```

Comment:
>Naast het eerder genoemde voordeel van herbruikbaarheid is een ander voordeel van het gebruik van autorisaties dat de patiënt dit ook inzichtelijk zou kunnen krijgen in een PGO. De patiënt zou op basis van de autorisaties een overzicht kunnen krijgen van wie zijn of haar gegevens kan inzien, en waarom.

Een ander voordeel van autorisaties is dat de ook de patient een overzicht in het PGO kan krijgen wie welke gegevens waarom kan inzien.